book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -41,6 +41,28 @@ docker run --rm -it --privileged -p 4000:4000 ^
 wallaroo-labs-docker-wallaroolabs.bintray.io/{{ docker_version_url }}
 {%- endcodetabs %}
 
+## Shell 1: Start the Wallaroo Docker container for Machida3
+
+{% codetabs name="UNIX Bash", type="bash" -%}
+docker run --rm -it --privileged -p 4000:4000 \
+-v /tmp/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/wallaroo-src:/src/wallaroo \
+-v /tmp/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/python-virtualenv-machida3:/src/python-virtualenv \
+--name wally \
+wallaroo-labs-docker-wallaroolabs.bintray.io/{{ docker_version_url }}
+{%- language name="Windows Powershell", type="bash" -%}
+docker run --rm -it --privileged -p 4000:4000 `
+-v c:/wallaroo-docker/wallaroo--{{ book.wallaroo_version }}/wallaroo-src:/src/wallaroo `
+-v c:/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/python-virtualenv:/src/python-virtualenv `
+--name wally `
+wallaroo-labs-docker-wallaroolabs.bintray.io/{{ docker_version_url }}
+{%- language name="Windows Command Prompt", type="bash" -%}
+docker run --rm -it --privileged -p 4000:4000 ^
+-v c:/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/wallaroo-src:/src/wallaroo ^
+-v c:/wallaroo-docker/wallaroo-{{ book.wallaroo_version }}/python-virtualenv:/src/python-virtualenv ^
+--name wally ^
+wallaroo-labs-docker-wallaroolabs.bintray.io/{{ docker_version_url }} -p python3
```

Comment:
Should the `-p python3` arg be added for non-`Windows Command Prompt` environments also?

Reply:
addressed